package.json

```diff
@@ -95,6 +104,7 @@
     "react-highlight": "briancappello/react-highlight#react-v16-compiled",
     "react-test-renderer": "^16.12.0",
     "rimraf": "^2.6.3",
+    "semantic-release": "^16.0.0-beta.43",
```

Comment:
@jamime: I see you added this comment in `rfcs/text/semantic-release.md`:

> The beta version of `semantic-release` includes extensive support for publishing a maintenance release. There is a detailed walk through outlining many scenarios. This functionality is not available in the latest stable release, however the beta has been evolving for in excess of a year.

I just wanted to check that we're all OK with using the beta version of `semantic-release` instead of the stable version?

Reply:
I have an example repo set up and I'm going to do some more testing regarding pre-release branches and patching to older releases. I did see https://github.com/stencila/dev-config/issues/27 which raises some concerns over the stability. However, the main reason for using the beta is to avoid having to migrate the tag information from v15 to v16 git notes format introduced in [beta 39](https://github.com/semantic-release/semantic-release/releases/tag/v16.0.0-beta.39)

More on this to follow...

Reply:
I'm happy with the beta release, I see no major issues raised on GitHub and I've tested the functionality described in the issue above and it works as intended.

Reply:
`v16.0.1` is now stable, I'll update the PR.